SayTheirNames/Source/Service/Service.swift

```diff
@@ -13,7 +13,8 @@ final class Service {
     lazy private(set) var navigator = Navigator(service: self)
```

Comment:
Related to my comment about the network being added in this class, this whole class feels like it could be a protocol. Probably out of scope of this PR, but it would really help us in testing these controllers with a `Servicing` protocol that defines `navigator`, `image`, `dateFormatter`, and `dataStore` (rather than `network`). The `Service` class could implement the protocol, and we could just pass around something that conforms to the protocol. 


Ok this is definitely out of scope for this PR.

Reply:
Completely agree. Sounds like a solid improvement for another PR. Haha